index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css ">
+        <script src="https://kit.fontawesome.com/5116129a4d.js"
+            crossorigin="anonymous"></script>
+        <title>AALA</title>
+    </head>
+
+    <body>
+
+        <section class="sec1">
+            <div class="desktopdarkheader">
+                <div class="dark-items">
+                    <ul>
+                        <li><i class="fa-brands fa-facebook-f"></i></li>
+                        <li><img src="images/twitter.png" alt=""></li>
```

Comment:
Kindly add a meaningful `alt text` to the `img` tag to follow the best practices 🤗

Reply:
done thanks